app/src/main/res/values/strings.xml

```diff
@@ -703,7 +703,11 @@ The new line here must be kept as the second half of the string is clickable for
 
     <!-- Tip displayed on the home view about the changes in the new release.
     %1$s is a placeholder for the app name -->
-    <string name="tip_fresh_look">Why yes, we do have a fresh new look!\n Read more about this and other updates to %1$s.</string>
+    <string name="tip_fresh_look">Why yes, we do have a fresh new look! Read more about this and other updates to %1$s.</string>
+
+    <!-- Tip displayed on the home view about the changes in the new release.
+   %1$s is a placeholder for the app name -->
+    <string name="tip_fresh_look2">Why yes, we do have a fresh new look!\n Read more about this and other updates to %1$s.</string>
```

Comment:
@Delphine To highlight just the second line of this tip I added a new string(tip_fresh_look2) with "\n". Can you tell me if you are ok with this change or I should revert it(I know that we are in a strings freeze)?

Reply:
So we updated that string on our side last week (more or less) but with no change in string ID. See example in Pontoon with French here!: https://pontoon.mozilla.org/fr/focus-for-android/mozilla-mobile/focus-android/app/src/main/res/values/strings.xml/?search=fresh&string=230880
I flagged it to localizers, but not all have updated their strings with that yet.
As I understand it, is the issue here that the link won't show up at all without the `/n`?
If that's the case, I think we should change the string ID as you suggest. I will make sure localizers are aware, if we do this

Reply:
Update: actually I see the link works from past conversations, but is not highlighted. Let me know if you want to update the string itself, or only add the `\n`

Reply:
Oh, but that's great, I didn't know that you already updated that string with "\n". Then I should revert my change(remove tip_fresh_look2) and let tip_fresh_look as it was

Reply:
Ok awesome! I have a list of locales that haven't updated that yet, so I will manually update those in our translation tool. It's not too much of a pain and that way we are sure it looks good across the board (and we don't break string freeze, yay!)
I guess the only thing you'd have to update on your side is adding it to en-US, unless that's good there.
Thanks for CCing me!

Reply:
All locales have been updated.
One thing I noticed though on the latest build I got is I could not even click that link - even for locales that already had the `\n` fix in such as French (cf [comment above](https://github.com/mozilla-mobile/focus-android/pull/5280#issuecomment-910695022))

Reply:
Yep, this pr should fix that